dipy/reconst/weights_method.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/python
+""" Functions for defining weights for iterative fitting methods."""
+
+import numpy as np
+
+MIN_POSITIVE_SIGNAL = 0.0001
+
+
+def simple_cutoff(residuals, log_residuals, pred_sig,
+                  design_matrix, leverages, C, cutoff):
+    """ Define outliers based on signal.
+    """
+    leverages[np.isclose(leverages, 1.0)] = 0.99  # avoids rare issues
+    HAT_factor = np.sqrt(1 - leverages)
+    cond = (np.abs(residuals) > + cutoff * C * HAT_factor)
+    return cond
+
+
+def two_eyes_cutoff(residuals, log_residuals, pred_sig,
+                    design_matrix, leverages, C, cutoff):
+    """ Two-eyes approach to define outliers, see [1]_.
```

Comment:
We have adopted [sphinxcontrib-bibtex](https://pypi.org/project/sphinxcontrib-bibtex/) for references.